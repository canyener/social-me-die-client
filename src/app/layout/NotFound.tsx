import React from 'react'
import { Link } from 'react-router-dom'
import { Button, Header, Icon, Segment } from 'semantic-ui-react'

const NotFound = () => {
  return (
    <Segment placeholder>
      <Header icon>
        <Icon name="search" />
      Oops! We've looked everywhere but couldn't find this.
    </Header>
      <Segment.Inline>
        <Button as={Link} to="/activities" primary>
          Return to Activities page.
      </Button>
      </Segment.Inline>
    </Segment>
  )
}

export default NotFound