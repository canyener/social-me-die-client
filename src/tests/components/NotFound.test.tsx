import { shallow } from 'enzyme'
import NotFound from '../../app/layout/NotFound'

test('Should render NotFound correctly', () => {
  const wrapper = shallow(<NotFound />)

  expect(wrapper).toMatchSnapshot()
})