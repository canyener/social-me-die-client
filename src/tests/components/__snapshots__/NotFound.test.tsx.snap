// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render NotFound correctly 1`] = `
<Segment
  placeholder={true}
>
  <Header
    icon={true}
  >
    <Icon
      as="i"
      name="search"
    />
    Oops! We've looked everywhere but couldn't find this.
  </Header>
  <SegmentInline>
    <Button
      as={
        Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        }
      }
      primary={true}
      to="/activities"
    >
      Return to Activities page.
    </Button>
  </SegmentInline>
</Segment>
`;
